File: src/main/java/com/dh/ondot/member/core/exception/TokenInvalidException.java
package com.dh.ondot.member.core.exception;

import com.dh.ondot.core.exception.UnauthorizedException;

import static com.dh.ondot.core.exception.ErrorCode.TOKEN_INVALID;

public class TokenInvalidException extends UnauthorizedException {

    public TokenInvalidException() {
        super(TOKEN_INVALID.getMessage());
    }

    @Override
    public String getErrorCode() {
        return TOKEN_INVALID.name();
    }
}

File: src/main/kotlin/com/dh/ondot/member/core/exception/TokenInvalidException.kt
package com.dh.ondot.member.core.exception

import com.dh.ondot.core.exception.ErrorCode.TOKEN_INVALID
import com.dh.ondot.core.exception.UnauthorizedException

class TokenInvalidException :
    UnauthorizedException(TOKEN_INVALID.message) {
    override val errorCode: String get() = TOKEN_INVALID.name
}
